packages/hardhat-core/src/internal/core/tasks/dsl.ts

```diff
@@ -15,10 +15,12 @@ import {
  * for creating and overriding tasks.
  */
 export class TasksDSL {
+  public readonly internalTask = this.subtask;
```

Comment:
I initially thought that this wouldn't work, as `this.subtask` is an unbound function. But calling it as `dsl.internalTask`  would re-bound it.

I think we should have a test for this alias.